frontend/src/reducer/notificationPage.js

```diff
@@ -65,38 +67,46 @@ export default function notificationPage (state = defaultNotificationsObject, ac
     }
 
     case `${ADD}/${NOTIFICATION}`: {
+      const notification = serializeNotification(action.notification)
+      const newUnreadMentionCount = notification.type === 'mention.created' ? state.unreadMentionCount + 1 : state.unreadMentionCount
       return {
         ...state,
         list: uniqBy([
-          serializeNotification(action.notification),
+          notification,
           ...state.list
         ], 'id'),
-        notificationNotReadCount: state.notificationNotReadCount + 1
+        unreadMentionCount: newUnreadMentionCount,
+        unreadNotificationCount: state.unreadNotificationCount + 1
       }
     }
 
     case `${READ}/${NOTIFICATION}`: {
       const notification = state.list.find(notification => notification.id === action.notificationId && !notification.read)
       if (!notification) return state
+      const newUnreadMentionCount = notification.type === 'mention.created' ? state.unreadMentionCount - 1 : state.unreadMentionCount
```

Comment:
Should we have a const for `'mention.created'`?

Reply:
I created a mentionCreated const